helm_deploy/hmpps-template-typescript/Chart.yaml

```diff
@@ -5,8 +5,8 @@ name: hmpps-template-typescript
 version: 0.2.0
 dependencies:
   - name: generic-service
-    version: 1.5.0
+    version: 2.1.0
```

Comment:
This major version bump was made backwards compatible in v2.1.0, though the change was to add the option of multiple **hosts** in the ingress instead of the fixed **host**. Therefor updated the values file as well.